frontend/src/pages/Graph/SummaryPanelCommon.tsx

```diff
@@ -121,7 +121,7 @@ export const getNodeMetrics = (
     case NodeMetricType.APP:
       return API.getAppMetrics(nodeData.namespace, nodeData.app!, options, nodeData.cluster);
     case NodeMetricType.SERVICE:
-      return API.getServiceMetrics(nodeData.namespace, nodeData.service!, options, nodeData.cluster);
+      return API.getServiceMetrics(nodeData.namespace, nodeData.service!, options);
```

Comment:
Are you sure we want to drop the 'cluster' here?  When you have selected a service node it is within a cluster (and within a namespace).  I think we may need to keep this discriminator.